### PR TITLE
CLI : Add a command line to add or remove a supervised entity (#7331)

### LINKED ISSUE #7331: Opfab cli : Add a command line to add or remove a supervised entity
opfab supervisor add-entity  ... 

opfab supervisor delete-entity ...

### PR DESCRIPTION
**Pull request by ClementBouvierN**
- In release notes :
  -  In chapter : Features
  -  Text : #7331 Opfab Cli : Add a command line to add or remove a supervised entity

### CODE REVIEW

File: cli/src/supervisorCommands.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+            default:
+                console.log(`Unknown supervisor command : ${command}
+                `);
+                await this.printHelp();
+                break;
+        }
+    },
+
+    async addSupervisorEntity(supervisedEntity, supervisingEntity) {
+        supervisedEntity = await utils.missingTextPrompt('Supervised entity name', supervisedEntity);
```
Comment (quinarygio) [inline cli/src/supervisorCommands.js:50]: Maybe it is better to check if supervisedEntity was set and if was not set then exit without asking for supervisor.

File: cli/src/supervisorCommands.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+        const entityToSuperviseResponse = await utils.sendRequest(
+            `supervisor/supervisedEntities`,
+            'GET',
+            undefined,
+            '',
+            `Failed to fetch entities to supervise`,
+            ``
+        );
+        if (!entityToSuperviseResponse.ok) {
+            return;
```
Comment (quinarygio) [inline cli/src/supervisorCommands.js:68]: Log an error message if entityToSuperviseResponse not ok

**Review by quinarygio — CHANGES_REQUESTED** [review]

File: cli/src/supervisorCommands.js (hunk 1 above)
Comment (ClementBouvierN) [reply cli/src/supervisorCommands.js:50]: Done

File: cli/src/supervisorCommands.js (hunk 2 above)
Comment (ClementBouvierN) [reply cli/src/supervisorCommands.js:68]: It is already logged by the utils.sendRequest() method calling it. The error message is specified in the call of the method

**Review by quinarygio — APPROVED** [review]

**Pull request merged**